src/server/repository/assessment/assessment/getCreateSchemaDDL.ts

```diff
@@ -357,9 +358,23 @@ export const getCreateOrReplaceViewCountryUserSummary = (props: { assessment: As
             ur.uuid,
             to_jsonb(ur.*) as role
         from users_role ur
-        where (ur.cycle_uuid = '${cycle.uuid}' and
-               ur.assessment_uuid = '${assessment.uuid}')
-           or ur.role = 'ADMINISTRATOR'
+        where (
```

Comment:
Update country user summaries to exclude atlantis roles for published cycles to avoid mismatch in logic
(E.g. Admin -> Collaborators: show atlantis country role for user, go to edit, the role is not visible)